main.py

```diff
@@ -1,7 +1,35 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Drama"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
+        ("George", "Clooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+    for first_name, last_name in actors:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
```

Comment:
There is a typo in the filter condition: 'Dramma' should be 'Drama'. This will prevent the update from executing correctly as there is no genre with the name 'Dramma'.